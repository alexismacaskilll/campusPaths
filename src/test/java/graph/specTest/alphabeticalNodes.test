# Tests adding nodes to a graph in reverse alphabetical order by node name and printing the
# nodes out in alphabetical order.

# Create a graph
CreateGraph graph1

# Add a node to the graph
AddNode graph1 n
# Add a node to the graph
AddNode graph1 m

# Print the nodes in the graph
ListNodes graph1