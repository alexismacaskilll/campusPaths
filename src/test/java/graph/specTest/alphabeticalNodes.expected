# Tests adding nodes to a graph in reverse alphabetical order by node name and printing the
# nodes out in alphabetical order.

# Create a graph
created graph graph1

# Add a node to the graph
added node n to graph1
# Add a node to the graph
added node m to graph1

# Print the nodes in the graph
graph1 contains: m n